CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change. 
+
+Please note we have a Code of Conduct, please follow it in all your interactions with the project.
+
+## Pull Request Process
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a 
+   build.
+2. Update the README.md with details of changes to the interface, this includes new environment 
+   variables, exposed ports, useful file locations and container parameters.
+3. Increase the version numbers in any examples files and the README.md to the new version that this
+   Pull Request would represent. The versioning scheme we use is [SemVer](http://semver.org/).
+4. You may merge the Pull Request in once you have the sign-off of two other developers, or if you 
```

Comment:
We currently don't tend to have more than one reviewer review these things (Mostly Raf reviewing my changes), so we would always be using the latter clause here. Maybe we continue as is and change it as needed?

Reply:
I agree to continue as is. We can enforce more "reviews" for breaking changes or external changes.